content/spin/v2/javascript-components.md

```diff
@@ -86,16 +77,17 @@ This creates a directory of the following structure:
 
 ```text
 hello-world/
+├── knitwit.json
 ├── package.json
-├── README.md
 ├── spin.toml
 ├── src
-│   └── index.ts
+│   ├── index.ts
+│   └── spin.ts
 ├── tsconfig.json
 └── webpack.config.js
 ```
 
-The source for the component is present in `src/index.ts`. [Webpack](https://webpack.js.org) is used to bundle the component into a single `.js` file which will then be compiled to a `.wasm` module using the `js2wasm` plugin.
+The source for the component is present in `src/index.ts`. [Webpack](https://webpack.js.org) is used to bundle the component into a single `.js` file which will then be compiled to a `.wasm` module using the `j2w` node executable provided by the `@fermyon/spin-sdk` which is a wrapper around `ComponentizeJS`. The `knitwit.json` is the configuration file used by [knitwit](https://github.com/fermyon/knitwit) to manage the WebAssembly dependencies of each package.
```

Comment:
Given the existing structure, we should treat this comment separately. But I'm not sure about including this kind of detail in the "installing templates" and "building your first app" section.

Reply:
Happy to remove that but not sure if it would be better to create a new heading explaining how the different pieces fit together.